lib/rules/indentation/__tests__/document-with-roots.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const postcssParse = require('postcss/lib/parse');
+const { stringify, Document, Input } = require('postcss');
```

Comment:
[suggestion] I think it's better to avoid internal importing like `require('postcss/lib/parse')` if possible. So how about refactoring like this?

```suggestion
const { parse: postcssParse, stringify, Document, Input } = require('postcss');
```